addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4761,7 +4762,33 @@ registry.sizing = SnippetOptionWidget.extend({
                 resizeResolve();
                 self.trigger_up("enable_loading_effect");
 
+                // Check whether there has been a resizing.
                 if (directions.every(dir => dir.begin === dir.current)) {
+                    const mouseupTime = new Date();
+                    // Mouse move duration in milliseconds.
+                    const mouseHeldDuration = mouseupTime - mousedownTime;
+                    // If there has been no resizing and the duration of the
+                    // mouse held has been less than 500 milliseconds. We assume
+                    // that the user wanted to click on the snippet behind the
+                    // handle (e.g. select an "Over The Content" header when it
+                    // is entirely covered by the handle of the first snippet
+                    // of the page).
+                    if (mouseHeldDuration < 500) {
+                        const clickEvent = new MouseEvent("click", {
+                            bubbles: true,
+                            cancelable: true,
+                            screenX: ev["screenX"],
+                            screenY: ev["screenY"],
+                        });
+                        const sameCoordinatesEls = self.ownerDocument
+                            .elementsFromPoint(ev["pageX"], ev["pageY"]);
```

Comment:
Nitpick but I prefer `ev.pageX` instead of `ev["pageX"]` :slightly_smiling_face: